.github/workflows/deployment.yml

```diff
@@ -55,10 +55,15 @@ jobs:
       - name: Run GitVersion
         id: gitversion
         uses: gittools/actions/gitversion/execute@v0.9.4
-      - name: Set up JDK
-        uses: actions/setup-java@v1
+      - name: Setup JDK 15 from jdk.java.net
```

Comment:
Setup -> Download

Reply:
This is not from https://github.com/sormuras/download-jdk#minimal-example-with-default-feature-and-using-environment-variables (is it?), thus I am fine with this change.